### PR TITLE
complile only f64

### PR DESCRIPTION
**Pull request by JohnBox**

### CODE REVIEW

File: src/f64.rs [hunk 1]
```diff
@@ -141,7 +136,7 @@ impl Float for F64 {
     }
 
     fn to_u32(&self, rnd: RoundingMode, exact: bool) -> u32 {
-        let ret = unsafe { softfloat_sys::f64_to_ui32(self.0, rnd.to_softfloat(), exact) };
+        let ret = unsafe { softfloat_sys::f64_to_ui64(self.0, rnd.to_softfloat(), exact) };
```
Comment (target-san) [inline src/f64.rs]: Why?

File: src/f64.rs [hunk 2]
```diff
@@ -151,7 +146,7 @@ impl Float for F64 {
     }
 
     fn to_i32(&self, rnd: RoundingMode, exact: bool) -> i32 {
-        let ret = unsafe { softfloat_sys::f64_to_i32(self.0, rnd.to_softfloat(), exact) };
+        let ret = unsafe { softfloat_sys::f64_to_i64(self.0, rnd.to_softfloat(), exact) };
```
Comment (target-san) [inline src/f64.rs]: Why?

File: Cargo.toml [hunk 3]
```diff
@@ -13,7 +13,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default              = ["8086-sse"]
+default              = ["8086"]
```
Comment (target-san) [inline Cargo.toml]: Please see my comments on `softfloat-sys` project PR

File: src/f64.rs (hunk 1 above)
Comment (JohnBox) [reply src/f64.rs]: bold hack around c99 types mismatch. fixed to correct type

File: src/f64.rs (hunk 2 above)
Comment (JohnBox) [reply src/f64.rs]: bold hack around c99 types mismatch. fixed to correct type

File: Cargo.toml (hunk 3 above)
Comment (JohnBox) [reply Cargo.toml]: answered there

**Pull request closed without merging**